assets/model_monitoring/components/src/model_monitor_feature_selector/run.py

```diff
@@ -29,7 +29,9 @@ def run():
     args = parser.parse_args()
 
     input_df1 = try_read_mltable_in_spark(args.input_data_1, "input_data_1")
-    input_df2 = try_read_mltable_in_spark(args.input_data_2, "input_data_2")
+    input_df2 = input_df1
```

Comment:
if input_df2 is none, then it use same value as input_df1? This is the logic change. If yes, maybe you can add this in line 41 and modify that comment.

Reply:
This PR trying to cover none check for baseline data as its optional. if either input available feature selection will work and in feature selection input1 is required